templates/press-centre/index.html

```diff
@@ -47,7 +47,7 @@ <h2 class="p-muted-heading">Canonical <br class="u-hide--small">at a glance</h2>
       </div>
       <div class="col-9 col-medium-4">
         <div class="p-block">
-          <h3 class="p-heading--2">Publishing the #1 OS in the cloud. <br class="u-hide--medium u-hide--small">Trusted by:</h3>
+          <h3 class="p-heading--2">Publishing the #1 OS in the cloud. <br class="u-hide--medium u-hide--small">Trusted by industry leaders.</h3>
```

Comment:
```suggestion
          <p class="p-heading--2">Publishing the #1 OS in the cloud. <br class="u-hide--medium u-hide--small">Trusted by industry leaders.</p>
```

The issue mentions that it should be changed to a paragraph

Reply:
done, also added the suru which was previously removed. This is a decision from the visual design meeting.